tests/unittest/disaggregated/test_cache_transceiver_single_process.py

```
@@ -798,123 +803,126 @@ def run_transfer_test(
+                for kv in ctx_kv_caches[rank]:
+                    kv.close()
+            for rank in range(gen_world):
+                for kv in gen_kv_caches[rank]:
+                    kv.close()
+
+    finally:
+        for tc in ctx_tcs + gen_tcs:
+            try:
+                tc.shutdown()
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)